mkdocs.yml

```diff
@@ -45,6 +45,8 @@ markdown_extensions:
   - admonition
   - footnotes
   - attr_list
+  - toc:
+      permalink: "#"
```

Comment:
This is only sorta related - it adds link anchors to all the ToC headings. Right now all the top-level headings don't have anchors.